idseq_dag/steps/run_srst2.py

```diff
@@ -69,6 +72,76 @@ def get_common_params(self):
         return ['--min_coverage', min_cov,'--threads', n_threads,
                 '--output',  os.path.join(self.output_dir_local, 'output'), '--log', '--gene_db', db_file_path]
 
+    def generate_mapped_reads_table(self):
```

Comment:
Can we rename this to `generate_mapped_reads_tsv`? This makes it more clear that this function is creating a file.